### PR TITLE
Update feature_store_kms_key_encryption.ipynb

### PR DESCRIPTION
**Pull request by paul-cb**
Fixed typo

*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sagemaker-featurestore/feature_store_kms_key_encryption.ipynb [hunk 1]
```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```
Comment (jkroll-aws) [inline sagemaker-featurestore/feature_store_kms_key_encryption.ipynb:11]: <p>Typo: <code>encerypt</code> -&gt; <code>encrypt</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (jkroll-aws) [inline sagemaker-featurestore/feature_store_kms_key_encryption.ipynb:11]: <p>Typos: <code>the the</code> -&gt; <code>the</code>	, <code>additonal</code> -&gt; <code>additional</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Comment by jkroll-aws** [comment]
Added a few more typos to fix as surfaced by the automated grammar check, thanks!

**Comment by zhaoqizqwang** [comment]
Closing this PR since it was created a long time ago and didn't pass all the checks. Feel free to open a new pull request instead.

**Pull request closed without merging**